syntax = "proto3";

package audio;

option go_package = "MMaxaMM.audio.v1;audiov1";

service AudioService {
    rpc Recognize (AudioRequest) returns (AudioResponse);
}

message AudioRequest {
    string URI = 1;
}

message AudioResponse {
    string content = 1;
}